CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [v2.1.4](https://github.com/ably/kafka-connect-ably/tree/v2.1.4)
+
+[Full Changelog](https://github.com/ably/kafka-connect-ably/compare/v2.1.3...v2.1.4)
+
+**Fixed bugs:**
+
+- Update ably-java version [\#111](https://github.com/ably/kafka-connect-ably/issues/111)
+
+**Merged pull requests:**
+
+- Bump ably java to 1.2.27 [\#112](https://github.com/ably/kafka-connect-ably/pull/112) ([AndyTWF](https://github.com/AndyTWF))
```

Comment:
A few things I think we should change:
- "ably java" should be "ably-java"
- "Update ably-java version" shouldn't be listed as a bug, I realise that the dep was updated to fix an underlying bug so it would be fine in that section if we actually specified the bug that was fixed, but just listing a dep update as a bugfix without any additional context sounds wrong to me
- I don't think it makes sense to list the issue and the PR as separate bullet points here because they both refer to the same change. Personally I would ditch the headings and go for something like:
```suggestion
- Bump ably-java to 1.2.27 [\#112](https://github.com/ably/kafka-connect-ably/pull/112) ([AndyTWF](https://github.com/AndyTWF))
```